ui/components/component-library/button-icon/button-icon.constants.js

```diff
@@ -2,5 +2,6 @@ import { Size } from '../../../helpers/constants/design-system';
 
 export const BUTTON_ICON_SIZES = {
   SM: Size.SM,
+  MD: Size.MD,
```

Comment:
We don't offer this in Figma so I'm doubtful this is the size used with the ButtonIcon button. Have reached out to @SaraCheikh to confirm

Reply:
This is what it looks like different icon sizes
Icon Sm

<img width="370" alt="Screenshot 2023-03-03 at 9 27 35 AM" src="https://user-images.githubusercontent.com/8112138/222787639-e47e4805-8bf7-4465-8c40-83251fce2bbc.png">

Icon Md (doesn't exist currently)

<img width="362" alt="Screenshot 2023-03-03 at 9 27 46 AM" src="https://user-images.githubusercontent.com/8112138/222787641-e9a7a2cb-d663-455a-9be2-057a7baa4ccb.png">

Icon Lg

<img width="365" alt="Screenshot 2023-03-03 at 9 28 02 AM" src="https://user-images.githubusercontent.com/8112138/222787643-1c6b05f6-5ef0-4cdf-b8bb-0807ee5f9263.png">

Reply:
As per slack conversation use `size={BUTTON_ICON_SIZES.LG}`